oauth2/oauth2-client/src/main/java/org/springframework/security/oauth2/client/web/reactive/result/method/annotation/OAuth2AuthorizedClientResolver.java

```diff
@@ -57,6 +61,10 @@
 
 	private String defaultClientRegistrationId;
 
+	private Clock clock = Clock.systemUTC();
+	private Duration accessTokenExpiresSkew = Duration.ofMinutes(1);
```

Comment:
We need a way to allow the user to configure `accessTokenExpiresSkew`

Reply:
Same as above: I saw the initial suggestion for this in a comment on the issue but when you responded that the skew already existed (as a fixed number), it looked like that addressed the concern, as the conversation went in a different direction. Are we thinking something as straightforward as a setter or did you have something else in mind? Just let me know.

Reply:
Same as above - please remove `accessTokenExpiresSkew` and it's associations